tests/test_port.py

```diff
@@ -6,6 +6,51 @@
 
 
 class TestPort(object):
+    def test_PortTpid(self, dvs, testlog):
+        pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+        # check application database for default TPID before set operation
+        pdb_port_tbl = swsscommon.Table(pdb, "PORT_TABLE")
+        (status, fvs) = pdb_port_tbl.get("Ethernet8")
+        assert status == True
+        tpid = "0x0000"
+        for fv in fvs:
+            if fv[0] == "tpid":
+                tpid = fv[1]
```

Comment:
By default, there is no `tpid `field in the port table until you set, correct? IMO, this check doesn't serve a purpose

Reply:
sure. let me take it out.